package.json

```diff
@@ -11,14 +11,69 @@
   ],
   "license": "AGPL-3.0",
   "author": "Rotki Solutions GmbH <info@rotki.com>",
-  "main": "index.js",
+  "main": "dist/index.umd.js",
+  "module": "dist/index.es.js",
+  "types": "dist/types/index.d.ts",
   "scripts": {
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix"
+    "postinstall": "pnpm dist:select",
+    "dist:select": "node scripts/dist-select.js",
+    "build:prod": "pnpm build:prod:v2 && pnpm build:prod:v3 && pnpm dist:select && pnpm build:types && pnpm build:tailwind",
+    "build:prod:v2": "vue-demi-switch 2.7 && vite build",
+    "build:prod:v3": "vue-demi-switch 3 && vite build",
+    "build:v2": "pnpm build:prod:v2 && pnpm run dist:select --force-version=2",
+    "build:v3": "pnpm build:prod:v3 && pnpm run dist:select --force-version=3",
+    "build": "pnpm build:v2",
+    "build-storybook": "storybook build",
+    "build:tailwind": "tailwindcss -o dist/style.css --minify",
+    "build:types": "vue-tsc -p tsconfig.build.json --declaration --emitDeclarationOnly --outDir dist/types",
+    "dev": "vite",
+    "lint": "eslint --cache . && pnpm run -r lint:style",
+    "lint:fix": "eslint --cache . --fix",
+    "storybook": "storybook dev -p 6006",
+    "typecheck": "vue-tsc --noEmit"
+  },
+  "peerDependencies": {
+    "vue": ">=2.7.14"
+  },
+  "dependencies": {
+    "@vueuse/core": "9.13.0",
+    "@vueuse/shared": "9.13.0",
+    "vue-demi": "0.13.11"
   },
   "devDependencies": {
+    "@babel/core": "7.21.3",
+    "@babel/types": "7.21.3",
     "@rotki/eslint-config": "1.0.3",
+    "@storybook/addon-essentials": "7.0.0-rc.8",
+    "@storybook/addon-interactions": "7.0.0-rc.8",
+    "@storybook/addon-styling": "0.3.2",
+    "@storybook/addon-links": "7.0.0-rc.8",
+    "@storybook/blocks": "7.0.0-rc.8",
+    "@storybook/testing-library": "0.0.14-next.1",
+    "@storybook/vue": "7.0.0-rc.8",
+    "@storybook/vue-vite": "7.0.0-rc.8",
```

Comment:
I setup storybook by their init script, but they install the rc version.
I will try to install the stable version and see if it's working.

Reply:
> I setup storybook by their init script, but they install the rc version. I will try to install the stable version and see if it's working.

I think 7 is still not stable, but it doesn't make sense to go with 6